notebooks/advanced_graphs.ipynb

```diff
@@ -569,7 +569,7 @@
     "* Two models\n",
     "\n",
     "The outlier detector is a special kind of transformer that will populate a tag in the response metadata with the outlier score it has calculated. \n",
-    "We use the docker image seldonio/mock_outlier_detector:1.0 for the outlier detector.\n",
+    "We use the docker image seldonio/outlier_mahalanobis:0.2 for the outlier detector.\n",
```

Comment:
Is it worth adding some explanation of the returned values from this test. Are  are the outlier scores meat to be useful here?

Reply:
No since the features sent are meaningless they can't really be interpreted

Reply:
actually, if you always send the same 2 points, as is the case with the rest_request, you will always see an outlier score of 0